src/seq/fasta.rl

```diff
@@ -0,0 +1,251 @@
+
+
+# FASTA sequence types
+
+@doc """
+Metadata for FASTA sequence records containing just a `description` field.
+""" ->
+type FASTAMetadata
+    description::String
```

Comment:
Sequences downloaded from RefSeq will have both name and description text, but real-world FASTA files often lacks this description text (e.g. Protein Data Bank).
So, I think wrapping ti with `Nullable` would be better.